dmutils/apiclient.py

```diff
@@ -64,13 +63,14 @@ def get_status(self):
         try:
             return self._get(
                 "{}/_status".format(self.base_url))
-        except APIError as e:
-            return e.response.json()
         except requests.RequestException as e:
-            return {
-                "status": "error",
-                "message": "{}".format(e.message),
-            }
+            try:
+                return e.response.json()
+            except ValueError:
+                return {
+                    "status": "error",
+                    "message": "{}".format(e.message),
```

Comment:
It's part of the standard interface of `Exception` so it should do. (for very large values of should)